src/ReverseProxy/Management/ProxyConfigManager.cs

```diff
@@ -66,7 +66,8 @@ internal sealed class ProxyConfigManager : EndpointDataSource, IProxyStateLookup
         ITransformBuilder transformBuilder,
         IForwarderHttpClientFactory httpClientFactory,
         IActiveHealthCheckMonitor activeHealthCheckMonitor,
-        IClusterDestinationsUpdater clusterDestinationsUpdater)
+        IClusterDestinationsUpdater clusterDestinationsUpdater,
+        IConfigNotifier? configNotifier = null)
```

Comment:
```suggestion
        IEnumerable<IConfigNotifier> configNotifiers)
```

Just like it's valid to have multiple `IClusterChangeListener`s, I think we should allow registering multiple config change notifiers.

Reply:
Fixed. I also slightly updated the test to ensure registration of multiple `ICconfigChangeNotifier`s works as expected